canonical-websites/www.ubuntu.com
# Update datasheet on /cloud/foundation-cloud

## Done

Update datasheet on /cloud/foundation-cloud

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/foundation-cloud](http://0.0.0.0:8001/cloud/foundation-cloud) or the [demo](http://www.ubuntu.com-pr-2429.run.demo.haus/cloud/foundation-cloud) 
- Make sure the datasheets in the hero and the second item of 'Get started' have been updated as per copy doc


## Issue / Card

Fixes: #2428 
Doc: https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#
